src/components/Filters/FilterFile/FilterFileForm.test.js

```diff
@@ -42,7 +42,6 @@ const renderFilterFileForm = (stripes, initialValues = FILTER) => {
                 stripes={stripes}
               >
                 <FieldArray
-                  addDocBtnLabel="Add file to filter"
```

Comment:
Test setup does not match actual code:
* `<FilterFileForm>` does not accept children
* `<FilterFileForm>` does not use the `stripes` prop